internal/operators/manager.go

```diff
@@ -95,19 +100,16 @@ func (mgr *Manager) ValidateHost(ctx context.Context, cluster *common.Cluster, h
 
 	results := make([]api.ValidationResult, 0, len(mgr.operators))
 
-	// To track operators that are disabled or not present in the cluster configuration, but have to be present
```

Comment:
Please restore the comment; I've been asked in my review to add it.

Reply:
Will do.